dom/src/test/java/org/apache/james/mime4j/message/HeaderImplTest.java

```diff
@@ -70,7 +75,7 @@ public void testWriteSpecialCharacters() throws Exception {
         byte[] b = outstream.toByteArray();
         ByteArrayBuffer buf = new ByteArrayBuffer(b.length);
         buf.append(b, 0, b.length);
-        String s = ContentUtil.decode(buf);
+        String s = ContentUtil.decode(StandardCharsets.UTF_8, buf);
```

Comment:
I would have the charset as a second argument.

Reply:
The method was already there, should it be changed?

Reply:
Then no I think